### DISCUSSION TITLE
Left/Right arrow keys not working in VS Code

### DISCUSSION THREAD

**Discussion by vfilter** [community/community#35532] [Copilot Conversations] [answered]
I'm using the latest VS Code version on a M1 Mac and cannot move my cursor with the left or right keys when Copilot is making a suggestion. Copilot nightlies same results. Up and down arrows work and things also work as soon as Copilot is disabled. Tried this in Rust and JS, same result, as long as there is a suggestion in the editor window, I cannot move left/right.

Couldn't find a bug report forum so here it is.

**Comment by iamcjwilliams** [comment]
@vfilter Our copilot team is investigating this issue

**Comment by vfilter** [comment, marked as answer]
Solution turned out was not Copilot itself but the AWS Toolkit VS Code extension that causes this behavior
